File: src/main/kotlin/com/yobi/standard/auth/redis/repository/TokenRepository.java
package com.yobi.standard.auth.redis.repository;

import com.yobi.standard.auth.redis.entity.Token;
import org.springframework.data.repository.CrudRepository;
import org.springframework.stereotype.Repository;

import java.util.Optional;

@Repository
public interface TokenRepository extends CrudRepository<Token, String> {

    Optional<Token> findByAccessToken(String accessToken);
}

File: src/main/kotlin/com/yobi/standard/auth/redis/repository/TokenRepository.kt
package com.yobi.standard.auth.redis.repository

import com.yobi.standard.auth.redis.entity.Token
import org.springframework.data.repository.CrudRepository
import org.springframework.stereotype.Repository

@Repository
interface TokenRepository : CrudRepository<Token, String> {
    fun findByAccessToken(accessToken: String): Token?
}
